src/FffGcodeWriter.cpp

```diff
@@ -747,24 +747,37 @@ void FffGcodeWriter::processRaft(const SliceDataStorage& storage)
         const coord_t max_resolution = base_settings.get<coord_t>("meshfix_maximum_resolution");
         const coord_t max_deviation = base_settings.get<coord_t>("meshfix_maximum_deviation");
 
-        Infill infill_comp(
-            EFillMethod::LINES, zig_zaggify_infill, connect_polygons, storage.raftOutline, gcode_layer.configs_storage.raft_base_config.getLineWidth(), line_spacing,
-            fill_overlap, infill_multiplier, fill_angle, z, extra_infill_shift,
-            max_resolution, max_deviation,
-            wall_line_count, infill_origin, skip_stitching, fill_gaps, connected_zigzags, use_endpieces, skip_some_zags, zag_skip_count, pocket_size
-            );
-        std::vector<VariableWidthLines> raft_paths;
-        infill_comp.generate(raft_paths, raft_polygons, raftLines, base_settings);
-        if(!raft_paths.empty())
+        std::vector<Polygons> raft_outline_paths;
+        if (storage.primeRaftOutline.area() > 0)
         {
-            const GCodePathConfig& config = gcode_layer.configs_storage.raft_base_config;
-            const ZSeamConfig z_seam_config(EZSeamType::SHORTEST, gcode_layer.getLastPlannedPositionOrStartingPosition(), EZSeamCornerPrefType::Z_SEAM_CORNER_PREF_NONE, false);
-            InsetOrderOptimizer wall_orderer(*this, storage, gcode_layer, base_settings, base_extruder_nr,
-                                             config, config, config, config,
-                                             retract_before_outer_wall, wipe_dist, wipe_dist, base_extruder_nr, base_extruder_nr, z_seam_config, raft_paths);
-            wall_orderer.addToLayer();
+            raft_outline_paths.emplace_back(storage.primeRaftOutline);
+        }
+        raft_outline_paths.emplace_back(storage.raftOutline);
+
+        for (const auto& raft_outline_path : raft_outline_paths)
```

Comment:
Going by the variable name, I'd think that this would be a variable-width path or a toolpath.

Writing down the type would clarify this. Alternatively, we can consider renaming it to `raft_outlines` or `raft_areas`.
```suggestion
        for (const Polygons& raft_outline_path : raft_outline_paths)
```